### ISSUE TITLE
Feature Request:  add a separate TAB to show NFT transactions only

### ISSUE THREAD

**Issue by paulpoco** [signum-network/signum-explorer#74]
Feature Request:  add a separate TAB to show NFT transactions only.

Include smart contract address, text name of NFT, bought or sold, changed price, etc

**Comment by pir8radio** [comment]
I agree with this...   NFT is basically a "token" so it should have its own tab with NFT holdings.

**Comment by rogerwim** [comment]
No, a NFT is a single app and we should not support those, the same Explorer should be general purpose.

**Comment by paulpoco** [comment]
Well right now the SignumArt portal Activities doesn't show all the full activities.   Thr portal says a NFT was Minted but not the contract/tx hash/names, same thing with buy or sell.

**Comment by pir8radio** [comment]
its a money transaction.   all transactions should be traceable on an explorer..   the explorer is what keeps the blockchain honest.   yea people can dig in the DB, but you dont dig in your banks DB to find tx's neither should users of signum.     I also agree there are holes in transactions..      I bought an NFT and transferred it to the nam pool account, and there is no trace that i was the original purchaser.

**Comment by paulpoco** [comment]
It is even more fun try to trace which NFT you actually sold.

My temp solution was to using bitcoin.tax keep track of my minting.  In comments for each 0.4 SIGNA minting in the tracking website what I did & name of NFT & contract for NFT & the hash of the tx ID.  Then I would get contract address of what was sold and search for that in my bitcoin.tax spending tab and list all the minting and search for the contract with the browser search and find the contract and therefore I would know the name of the NFT sold.

Right now the Explorer is jumping around from page to page almost confusing.

**Comment by rever273** [comment]
I support this proposal, SRC40 should have a separate tab

**Comment by pir8radio** [comment]
there is a loop hole that signumart.io wont show when looking up a user at https://www.signumart.io/item/11055356809051900004           If i buy an NFT and then transfer it to someone else, if they have not created a signumart.io account you cant track the nft's they own without going into each and every NFT to see if they own it.  Its a way to hide NFT ownership AKA hide money.    needs to be in the explorer some how.
